transcom/mymove
# B-19728-INT customer upload additional docs

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19728)
## [FF Repo PR](https://github.com/transcom/milmove-feature-flags/pull/160)

## Summary

Give the customers the ability to upload documents without affecting their workflow!

This work create a new button for customers that appears on their MoveHome page after a move is submitted, that allows them to provide additional documentation to the government without affecting their workflow. We added a new column to the moves table that houses additional_documents_id. Customer have the ability to upload documents and delete documents after they've been uploaded.

### How to test

1. Login as a customer with a move that has been submitted, and navigate to the MoveHome page of a move.
2. Click on the Upload Additional Documents button attached to the Profile step.
3. Confirm the page looks good and the upload/delete works, and persists as you navigate away/log out
4. Create new moves and add documents to new moves and previous moves, and make sure they're attaching to the correct move
5. Try and break it
